sqld/src/http/mod.rs

```diff
@@ -250,7 +255,7 @@ where
         }
     });
 
-    server.await?;
+    server.await.map_err(Error::HttpServerError)?;
```

Comment:
this is one instance where anyhow is fine. This error will never be exposed to the client, but will cause the app to exit. We'll never need to give it an error code.